<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecisionMarketCreationWidget</class>
 <widget class="QWidget" name="DecisionMarketCreationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMarketType">
        <property name="text">
         <string>Market Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonStandard">
        <property name="text">
         <string>Standard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDAC">
        <property name="text">
         <string>DAC (Selling Disabled)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelHashFunction">
        <property name="text">
         <string>Hash Function:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxHashFunction">
        <item>
         <property name="text">
          <string>SHA-2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDifficulty">
        <property name="text">
         <string>Difficulty</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxDifficulty">
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMaturation">
        <property name="text">
         <string>Maturation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxMaturation">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelBranch">
     <property name="text">
      <string>Branch:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxBranch">
     <item>
      <property name="text">
       <string>Main</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sports</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Econ</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSelectDecision">
     <property name="text">
      <string>Select Decision</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelB">
        <property name="text">
         <string>Liquidity Factor:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxB">
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMaxCommission">
        <property name="text">
         <string>Max Commission:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxCommission">
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTradeFee">
        <property name="text">
         <string>Trading Fee:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxTradeFee">
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAuthorAddress">
        <property name="text">
         <string>Author Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAuthorAddress"/>
      </item>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTitle"/>
      </item>
      <item>
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
      </item>
      <item>
       <widget class="QLabel" name="labelTags">
        <property name="text">
         <string>Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTags"/>
      </item>
      <item>
       <widget class="QLabel" name="labelDecisions">
        <property name="text">
         <string>Decisions:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditDecisions"/>
      </item>
      <item>
       <widget class="QLabel" name="labelFunctions">
        <property name="text">
         <string>Functions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFunctions">
        <item>
         <property name="text">
          <string>X1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LNX1</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCreateMarket">
        <property name="text">
         <string>Create Market</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonUpdateMarket">
        <property name="text">
         <string>Update Market</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
